pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>xxxxxx.yyyyyy.zzzzzz</groupId>
     <artifactId>projectName</artifactId>
-    <version>5.5.0-SNAPSHOT</version>
+    <version>5.5.0.RC2-SNAPSHOT</version>
```

Comment:
This comment is not related with this modification.

- [ ] Please fix errors at application starting associated with https://github.com/terasolunaorg/terasoluna-gfw/pull/824#issuecomment-397950136

Reply:
@hirakuriy 
Fixed on below:
bbc2c2c8b94a6669e408adfdce93200c950b9c40

Reply:
@hirakuriy OK, please approve.